src/main/kotlin/com/toasttab/pulseman/state/Tab.kt

```diff
@@ -1,159 +1,23 @@
-/*
- * Copyright (c) 2021 Toast Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
```

Comment:
The copyright notice was removed, is that wanted? It seems to be in all other files also the permissions of the file changed?

Reply:
Great thanks.